content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -721,12 +721,7 @@ The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
 
 ### Matching requests: `matchConditions`
 
-{{< feature-state state="alpha" for_k8s_version="v1.27" >}}
-
-{{< note >}}
-Use of `matchConditions` requires the [featuregate](/docs/reference/command-line-tools-reference/feature-gates/)
-`AdmissionWebhookMatchConditions` to be explicitly enabled on the kube-apiserver before this feature can be used.
-{{< /note >}}
+{{< feature-state state="beta" for_k8s_version="v1.28" >}}
 
 You can define _match conditions_for webhooks if you need fine-grained request filtering. These
```

Comment:
Not changed by this PR but this in the document causes it be rendered incorrectly

```suggestion
You can define _match conditions_ for webhooks if you need fine-grained request filtering. These
```